.NET/Microsoft.Recognizers.Definitions.Common/BaseCurrency.cs

```diff
@@ -268,5 +268,16 @@ public static class BaseCurrency
             { @"Hào", 10 },
             { @"Ngwee", 100 }
         };
+      public static readonly Dictionary<string, long> SpecialCurrencyFractionMax = new Dictionary<string, long>
+        {
+            { @"JOD", 1000 },

```

Comment:
You can remove the CYP from this list. For all purposes it can be considered as 100 fractional, as that was it's base when abolished.